### ISSUE TITLE
About logging  problem

### ISSUE THREAD

**Issue by yuanyuan1993** [eriklindernoren/PyTorch-YOLOv3#246]
Continue training the last saved model. How can you continue to output logs on the last saved log？ Thanks~

**Comment by www12345678** [comment]
I have a question, training interruption, how to use the training model to continue training?

**Comment by yuanyuan1993** [comment]
> I have a question, training interruption, how to use the training model to continue training?
python3 train.py --model_def config/yolov3-yuan.cfg --data_config config/yuan.data   --pretrained_weights checkpoints/yolov3_ckpt_299.pth  --epoch 300
you   can  use " --pretrained_weights checkpoints/yolov3_ckpt_299.pth "  to any training model to continue training

**Comment by yyx1107** [comment]
> I have a question, training interruption, how to use the training model to continue training?

change the pretrained weights and the start epoch (if the learning rate is changing with the iteration )
